requirements/runtime.txt

```diff
@@ -3,5 +3,6 @@
 
 https://github.com/level12/keg/zipball/master
 arrow
+wtforms_components
```

Comment:
1. Unless you have a direct dependency on this in our project, it should not be listed. It's a dependency of WTForms-Alchemy.
2. You didn't specify it correctly.  If it is needed here, it should be: WTForms-Components.